app/components/views/GetStartedPage/OpenWallet/CreateWalletForm/ConfirmSeed/Form.js

```diff
@@ -20,31 +26,55 @@ class ConfirmSeedForm extends React.Component{
     });
   }
 
+  handleToggle = (side) => {
+    this.setState({ seedType: side === "left" ? "words" : "hex"});
+  }
+
   render(){
-    const { remainingSeedWords, setSeedWords, isMatch, seedError, isEmpty } = this.props;
+    const { isMatch, seedError, isEmpty, seedWords, setSeedWords, createWalletExisting } = this.props;
+    const { seedType } = this.state;
+    const remaining = getRemaining(seedWords, seedType);
     return (
       <div className="confirm-seed">
-        <div className="create-wallet-label">
-          <div className="confirm-seed-label-text">
-            <T id="confirmSeed.label" m="Confirm Seed" />:
-            <InfoModalButton
-              modalTitle={<h1><T id="confirmSeed.seedInformation" m="Seed information" /></h1>}
-              modalContent={<SeedInfoModalContent />}
-            />
+        <div className="create-wallet-header">
+          <div className="create-wallet-label">
+            <div className="confirm-seed-label-text">
+              <T id="confirmSeed.label" m="Confirm Seed" />:
+              <InfoModalButton
+                modalTitle={<h1><T id="confirmSeed.seedInformation" m="Seed information" /></h1>    }
+                modalContent={<SeedInfoModalContent />}
+              />
+            </div>
+            <div className="confirm-seed-label-remaining-words">
+              {seedType === "words" ?
+                <T id="confirmSeed.wordsRemaining" m="{remaining, plural, one {one word remaining} other {# words remaining} }"
+                  values={{remaining: remaining}} />
+                :
+                <T id="confirmSeed.hexSymbolsRemaining" m="{remaining, plural, one {one hex symbol: your key must have at least 32 symbols and 128 max} other {# hex symbols: your key must have at least 32 symbols and 128 max } }"
```

Comment:
Can you explain the 32 min 128 max?  Afaik it will always be 32 bytes (64 chars when written out)

Reply:
I thought so too, but while coding, depending on the seed I was entering I notice the following `errors` or `warning` (that I am pretty certain were coming from the backend `decodeSeed` method)

- Warning 1:  Would be shown in case a no `HEX` symbol was entered, which makes sense, so it only accepts 0-F
- Warning 2: Would appear if the seed has an odd length
- Warning 3: Would say that the seed must have between 128 and 512 bytes, therefore a minimum of 32 and a max of 128 symbols

I will post the exactly messages as soon as I can, but that is pretty much does reason.

am I missing something?

Reply:
Yah those warnings are coming from dcrd/hdkeychain:https://github.com/decred/dcrd/blob/master/hdkeychain/extendedkey.go#L457-L459

So you "can" make a wallet with a seed that is between 16 and 128 bytes, but we only provide 32 byte seeds from all of our software.  I suppose we should allow people to use any length between 16 and 128.   Let me discuss with the other guys.

Reply:
And could you also add input validation to only accept 0-F for the hex?

Reply:
Also in terms of input validation, probably best to not send to wallet to decode the seed until it is > 32 chars and not odd.

Reply:
OK so confirmed that we're OK with > 16 and < 128.  But maybe add a warning on seeds that aren't 32 bytes that these come from non-supported software and may have unintended consequences.

Reply:
should we allow the same thing for the `words` input type?

Reply:
No, leave the words input as is.